docs/extensibility.md

```diff
@@ -1,23 +1,22 @@
 # Extensibility
-While Akri has several [currently supported discovery protocols](./roadmap.md#currently-supported-protocols) and sample brokers and applications to go with them, the protocol you want to use to discover resources may not be implemented yet. This walks you through all the development steps needed to implement a new protocol and sample broker. It will also cover the steps to get your protocol and broker[s] added to Akri, should you wish to contribute them back. 
 
-To add a new protocol implementation, three things are needed:
-1. Add a new DiscoveryHandler implementation in the Akri Agent
-1. Update the Configuration CRD to include the new DiscoveryHandler implementation
-1. Create a protocol broker for the new capability
+While Akri has several [currently supported discovery protocols](./roadmap.md#currently-supported-protocols) and sample brokers and applications to go with them, the protocol you want to use to discover resources may not be implemented yet. This walks you through all the development steps needed to implement a new protocol and sample broker. It will also cover the steps to get your protocol and broker[s] added to Akri, should you wish to contribute them back.

```

Comment:
I wonder if we should clarify the terms/interchangeability of the terms "discovery handler" and "protocol" here. Maybe "This walks you through all the development steps needed to implement a new protocol/discovery handler and sample broker. More broadly, Akri defines the procedure used to discover a certain set of devices a "discovery handler". While a discovery handler could be a protocol implementation (like Zeroconf or OPC UA) it could also be something other than a protocol like udev,"

Reply:
Hmm but then later on a "protocol implementation" consists of a discovery handler and more. I am still trying to wrap my head around the terms

Reply:
i'm happy to change this to whatever clarifies it for you :)

Reply:
Maybe just a 
> **Note** a protocol implementation can be any set of steps to discover devices. It does not have to be a "protocol" in the traditional sense. For example, Akri defines udev (not often called a "protocol") and OPC UA as protocols.